habitat-lab/habitat/gui/replay_gui_app_renderer.py

```diff
@@ -112,14 +112,32 @@ def render_update(self, dt):
 
         self._replay_renderer.render(mn.gl.default_framebuffer)
 
-        # arrange debug images on right side of frame, tiled down from the top
-        dest_y = self.window_size.y
-        for image in self._debug_images:
-            im_height, im_width, _ = image.shape
-            self._image_drawer.draw(
-                image, self.window_size.x - im_width, dest_y - im_height
-            )
-            dest_y -= im_height
+        if len(self._debug_images):
```

Comment:
This change is adding text labels to debug images. `_debug_image[i]` is assumed to be tuple `(title, image_tensor)`.